README.md

```diff
@@ -10,12 +10,30 @@ Install dependencies using:
 ```bash
 yarn install
 ```
-Run client using:
+
+Run the client using:
 ```bash
 yarn start
 ```
 The proxy should now be available at `http://localhost:4040`
 
+## Development
+
+Instead, run the client using:
+```bash
+yarn start-dev
+```
+
+The proxy will again be at `http://localhost:4040`, but nodemon will reload the proxy when changes are made.
+
+## Docker
+
+There is a supported docker image at ghcr.io/vector-im/trafficlight-proxy can be used to run this proxy.
+
+You must pass `TRAFFICLIGHT_URL` and `PROXY_URL`.
+
+`PROXY_URL` should be set to an endpoint that the trafficlight clients can access. At present there is no way for the proxy to bind to a port different to the URL that clients would connect to; however the hostname is ignored.
```

Comment:
Tracked at https://github.com/vector-im/trafficlight-proxy/issues/3

For /login and well known endpoints, we replace any occurrences of the homeserver URL with that of the proxy URL; so the hostname is not completely ignored.